debugging-via-dotnet-dump/test.sh

```diff
@@ -3,9 +3,8 @@
 # Check whether dotnet-dump, and its subcommands like `ps`, `collect`
 # and `analyze` are working
 
-if [ -f /etc/profile ]; then
-  source /etc/profile
-fi
+# Ensure global tools are on PATH.
+export PATH=~/.dotnet/tools:$PATH
```

Comment:
Note to self: we should a test to make sure /etc/profile adds dotnet tools to path.

Reply:
https://github.com/redhat-developer/dotnet-regular-tests/pull/255